### DISCUSSION TITLE
Can we use a specific version of Karpenter to install ?

### DISCUSSION THREAD

**Discussion by subashprabanantham** [aws-ia/terraform-aws-eks-blueprints-addons#211] [Q&A] [answered]
Hi,

I was trying out eks-blueprints-addons for my eks cluster setup. I was wondering where/how we fetch the version for each addon and is there a way to override it ?

Example:

I am using Karpenter, I would like to know what is the version and how to update it ?

**Comment by askulkarni2** [comment, marked as answer]
Hi @subashprabanantham you can view the version within the respective add-on's module in `main.tf`. E.g. https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/blob/c4c821c8f1c78982a781ca898770ce5da6c07ea0/main.tf#L2634

You can pass a different version like this.

```hcl
enable_karpenter = true 
karpenter = {
  chart_version  = "0.29.2"
}
```

**Reply by subashprabanantham** [reply]
Thanks @askulkarni2 for the quick reply!
